src/components/rangeSelect.tsx

```diff
@@ -0,0 +1,69 @@
+import React, { FC } from "react"
+
+import Select from "./select"
+
+interface Props {
+  name: {
+    min: string
+    max: string
+  }
+  handleChangeMin: (value) => void
+  handleChangeMax: (value) => void
+  value?: {
+    min: number
+    max: number
+  }
+  isValid?: {
+    min: boolean
+    max: boolean
+  }
+  placeholder?: {
+    min: string
+    max: string
+  }
+  required?: boolean
+  options?: Array<{ name: string; value: number | { customValue: number } }>
+}
```

Comment:
Would select not need a label too?

Reply:
From my understanding, this component is used only in listings as a filter and is wrapped within 
`<CollapsibleFilter
      title={t("listing.seats.label")}
      isCollapsed={true}
      mobileOnly={mobileOnly}
      name="seats"
      key="seats"
    >` which returns the label. 
    Writing this, I realised that we did not really need to extract this component since it is a not a shared one. What do you think ?

Reply:
I'd say let's not extract it at the moment then. We should've split the ticket per component anyway.